apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx

```diff
@@ -426,7 +426,12 @@ export const OrganizationHeader: React.FC<
         />
       )
     case 'nti':
-      return (
+      return n('usingDefaultHeader', false) ? (
+        <DefaultHeader
+          {...defaultProps}
+          image="https://images.ctfassets.net/8k0h54kbe6bj/eXqcbclteE88H5iQ6J3lo/bbc1d0c9d3abee93d34ec0aa718c833b/Group__1_.svg"
```

Comment:
I wonder if we should just in case read this from a namespace and have this as the fallback in case the image gets deleted from the cms in the future and the url dies